packages/core/src/modules/dif-presentation-exchange/utils/transform.ts

```diff
@@ -32,8 +34,10 @@ export function getSphereonOriginalVerifiablePresentation(
     verifiablePresentation instanceof W3cJsonLdVerifiablePresentation
   ) {
     return verifiablePresentation.encoded as SphereonOriginalVerifiablePresentation
-  } else if (verifiablePresentation instanceof MdocVerifiablePresentation) {
-    throw new CredoError('Mdoc verifiable presentation is not yet supported by Sphereon.')
+  } else if (verifiablePresentation instanceof MdocDeviceResponse) {
+    return com.sphereon.mdoc.data.device.DeviceResponseCbor.Static.cborDecode(
+      new Int8Array(TypedArrayEncoder.fromBase64(verifiablePresentation.base64Url))
+    )
```

Comment:
original can just be a string for PEX / ssi-types now. That way we do'nt have to depend on / import the sphereon mdl lib